suite-native/module-settings/src/components/ViewOnly/WalletRow.tsx

```diff
@@ -90,6 +97,8 @@ export const WalletRow = ({ device }: { device: TrezorDevice }) => {
         });
     };
 
+    const showToggleButton = selectedDevice?.state !== device.state || !isDiscoveryActive;
```

Comment:
Why this part of the condition? `selectedDevice?.state !== device.state ||` Why not only `!isDiscoveryActive`?

Reply:
So you can toggle View only for other devices (those not connected).

Reply:
Otherwise it would remove the button for all wallets in case this wallet was in the discovery process

Reply:
I see, what about 
```suggestion
    const showToggleButton = device.remember || !isDiscoveryActive;
```
?

I think it is simpler and even more correct - with passhprase you can have multiple devices with unfinished discovery.

Reply:
This probably wont work every time:
- connect device, wait for discovery
- remember device and disconnect it
- connect another device, wait until discovery starts
- switch to portfolio manager
- go to view only settings
- see this:

![Simulator Screenshot - iPhone 15 - 2024-05-22 at 13 18 38](https://github.com/trezor/trezor-suite/assets/2011829/8bcecb7d-634f-49b5-8dc7-e2c8a70d0a89)